frontend/test/__support__/server-mocks/search.ts

```diff
@@ -5,12 +5,24 @@ import type {
   Collection,
   Table,
   Database,
+  SearchModelType,
 } from "metabase-types/api";
 
-type SearchItem = Card | Dashboard | Collection | Table | Database;
+type SearchItem =
+  | Card
+  | Dashboard
+  | Collection
+  | Table
+  | (Database & {
+      collection: Record<string, any>;
+    });
```

Comment:
How can a database have a `collection` here?

Reply:
🤷  ask whoever wrote the search api. it's necessary for the search results component to work correctly.

![Screen Shot 2023-02-17 at 11 32 09 AM](https://user-images.githubusercontent.com/30528226/219752986-4d5de599-8492-44e5-b3bb-b042b3f58ad2.png)